src/EnergyPlus/DataBranchNodeConnections.hh

```diff
@@ -71,58 +71,58 @@ namespace DataBranchNodeConnections {
         std::string InletNodeName;  // Inlet Node ID
         std::string OutletNodeName; // Outlet Node ID
         std::string Description;    // Description of Component List Type
-        bool InfoFilled;            // true when all information has been filled
 
         // Default Constructor
-        ComponentListData() : InfoFilled(false)
-        {
-        }
+        ComponentListData() = default;
     };
 
     struct NodeConnectionDef
     {
         // Members
         int NodeNumber;                                // Node number of this node connection
         std::string NodeName;                          // Node Name of this node connection
-        std::string ObjectType;                        // Object/Component Type of this node connection
+        DataLoopNode::ConnectionObjectType ObjectType; // Object/Component Type of this node connection
         std::string ObjectName;                        // Name of the Object/Component Type of this node connection
-        std::string ConnectionType;                    // Connection Type (must be valid) for this node connection
+        DataLoopNode::ConnectionType ConnectionType;   // Connection Type (must be valid) for this node connection
         NodeInputManager::CompFluidStream FluidStream; // Fluid Stream for this node connection
         bool ObjectIsParent;                           // Indicator whether the object is a parent or not
 
         // Default Constructor
-        NodeConnectionDef() : NodeNumber(0), FluidStream(NodeInputManager::CompFluidStream::Invalid), ObjectIsParent(false)
+        NodeConnectionDef()
+            : NodeNumber(0), ObjectType(DataLoopNode::ConnectionObjectType::Invalid), ConnectionType(DataLoopNode::ConnectionType::Invalid),
+              FluidStream(NodeInputManager::CompFluidStream::Invalid), ObjectIsParent(false)
```

Comment:
Next time just add initialization at the variable declarations and get rid of this default constructor.  Definitely not holding it up for this though.